sycl/ReleaseNotes.md

```diff
@@ -1,52 +1,687 @@
+- Fixed a bug preventing sanitizers to be used together with shared libraries.
+  intel/llvm#18819
+- Fixed some memory leaks in sanitizers implementation. intel/llvm#19066
+- Fixed possible hangs when an application is stopped by a sanitizer because
+  errors were discovered. intel/llvm#19085
+
+- 97f508bd257d [DevSAN] Set maximum supported local/private shadow memory size (#19465)
+  -  What if the limit we set is too small? What is a user-visible side effect of this?
+
+- 69c3b9f9e01b [SYCL] Cherry-pick sanitizer patches (#19867)
+  -  Looks like 2/3 were fixes for regressions, as such the plan is to omit this
+- fae8cc2630b5 [DeviceSanitizer] Print log when build/link fail (#17521)
+- 55416cae415f [DeviceSanitizers] Set names to nameless globals (#17599)
+
+#### Address sanitizer
+
+- Improved error messages which are logged if enqueue kernel failed with
+  address sanitizer enabled. intel/llvm#20237
+- Fixed a memory leak and potential undefined behavior that may happen in
+  address sanitizer when application exits. intel/llvm#18612
+
+#### Thread sanitizer
+
+- Fixed a bug in thread sanitizer which may result in false positives reported
+  on subsequent kernel launches. intel/llvm#19355
+- Improved thread sanitizer performance by avoiding unnecessary device -> host
+  synchronization if there are no reports to transfer. intel/llvm#19133
+- Added support for detecting data races in local memory. intel/llvm#18718
+- Refined thread mapping algorithm for better performance of the thread
+  sanitizer. intel/llvm#19010
+- Added support for detecting data races in SYCL buffers. intel/llvm#17625
+- Fixed false positive reports coming from joint matrix usage. intel/llvm#18773
+- Added support for thread sanitizer in AOT mode. intel/llvm#18130
+- Fixed false positive reports coming from barriers usage. intel/llvm#17900
+- Fixed memory leaks in thread sanitizer. intel/llvm#18880
+- Improved thread sanitizer performance. intel/llvm#19127
+
+- 1d7e9ef8e71a [DevTSAN] Fix missing intercepted API and release local shadow (#18942)
+  -  Looks like some bugfix
+- e2968564552a [DevTSAN] Don't insert cleanup instruction for dynamic allocas (#18090)
+  -  Unclear what this is
+- 597709bd52c0 [DevTSAN] Treat each work item as a thread for both CPU & GPU device (#18580)
+- 1a83e6ec3272 [DevTSAN] Treat each work item as a thread for GPU device (#18347)
+  -  ???
+
+#### Memory sanitizer
+
+- Enabled origin tracking which provides information about where a memory
+  allocation associated with an issue report comes from. intel/llvm#18693
+- Fixed a bug with `async_work_group_copy` causing false negative report.
+  intel/llvm#18216
+
+- b07a0cf68226 [DevMSAN] Always make sure poison shuffle mask point to clean shadow (#18191)
+  -  Some bugfix?
+- 923eca9baf7c [DevMSAN] Don't report error when write `__MsanLaunchInfo` failed (#18129)
+  -  Some bugfix?
+- 23969c664d94 [DevMSAN] Fix wrong arguments when calling Memset (#19188)
+  -  Not sure what does it fix
```

Comment:
It's to fix a bug that could cause a segfault if kernel used static local memory.